shell/common/rasterizer.cc

```diff
@@ -22,6 +22,15 @@ namespace flutter {
 // used within this interval.
 static constexpr std::chrono::milliseconds kSkiaCleanupExpiration(15000);
 
+// TODO(dnfield): Remove this once internal embedders have caught up.
```

Comment:
I'm curious if the internal embedders have already caught up? (Today I just found an ancient note saying that we can remove this once https://github.com/flutter/engine/pull/9484/ is landed... But I couldn't remember why that's the case...)

Reply:
We should be good. The concerns were fuchsia, which is now in tree for us, and someone else who wasn't yet on the emvedder API I believe who now is